Formula/yarn.rb

```diff
@@ -1,26 +1,19 @@
-require "language/node"
-
 class Yarn < Formula
   desc "JavaScript package manager"
   homepage "https://yarnpkg.com/"
   url "https://yarnpkg.com/downloads/0.21.3/yarn-v0.21.3.tar.gz"
```

Comment:
You could keep the mirror just in case `yarnpkg.com` goes down, but the Windows Chocolatey package also uses yarnpkg.com and I haven't seen any reports of being unable to hit yarnpkg.com.